CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Add version bumping script for Craft (#1518)
```

Comment:
maybe we use skip-changelog for such changes? the changelog is meant for users facing changes.